README.md

```diff
@@ -20,10 +20,10 @@ This plugin uses [terser](https://github.com/terser-js/terser) to minify your Ja
 
 If you are using webpack v5 or above you do not need to install this plugin. Webpack v5 comes with the latest `terser-webpack-plugin` out of the box.
 
```

Comment:
Please describe it here

Reply:
Thank you for your answer. Do you mean I have to (1) additionally describe it here **OR** (2) remove what I suggested and add it there only?

Suggestion:

> If you are using webpack v5 or above you do not need to install this plugin. Webpack v5 comes with the latest `terser-webpack-plugin` out of the box. Using Webpack v4, you have to install `terser-webpack-plugin` v4.

Reply:
No need to change original setup, let's describe separately about webpack@4, anyway I strong recommend to migrate on v5

Reply:
@alexander-akait I just removed my original suggestion and tried to follow your advice. Let me know if it's OK!

Reply:
All is OK :+1: